web/src/resize.ts

```diff
@@ -191,8 +191,6 @@ export function resize_sidebars(): void {
 
 export function update_recent_view(): void {
     const $recent_view_filter_container = $("#recent_view_filter_buttons");
-    const recent_view_filters_height = $recent_view_filter_container.outerHeight(true) ?? 0;
-    $(":root").css("--recent-topics-filters-height", `${recent_view_filters_height}px`);
```

Comment:
Looks like 371cd0da6cc4da913d05c0936a6e32458daa652b is what removed it. You can find this sort of history really quickly with `git log -Srecent-topics-filters-height`, since the last reference will almost always be the PR that removed the usage. @amanagr FYI.